### ISSUE TITLE
A11y chip active vs inactive

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2272]
## Description

The color is the only way to differentiate an active or inactive chip. This is a failure of use of color.

https://www.w3.org/WAI/WCAG22/Understanding/use-of-color.html

### Bench

Material Design replaces the leading icon in filter chips with a check icon to indicate the active state.

### Workaround

BF has been updated to suggest users use the check icon in filter chips:

https://brandfactory.telefonica.com/document/1846#/components/chips/accessibility

## Tasks

- [ ] Update specs
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [ ] Android ticket
- [ ] iOS ticket
- [ ] Web ticket

**Comment by yceballost** [comment]
# Discussion
https://github.com/Telefonica/mistica-design/discussions/2218
